### PR TITLE
adds docs for media android end user client apps

### PR DESCRIPTION
**Pull request by elear-akshay**
1. adds terminology
2. pre-install checklist
3. download and install
4. quickstart guide
5. sample app walkthrough
6. api reference docs

### CODE REVIEW

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst [hunk 1]
```diff
@@ -3,12 +3,390 @@
 Quick Start Guide
```
Comment (manavkumarm) [inline source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst:5]: Need to swap the content here with the Sample App Walkthrough similar to PR # 8

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/sample_app_walkthrough_android_media_client_apps.rst [hunk 2]
```diff
@@ -3,9 +3,62 @@
...
+   - Visit :ref:`guide for service apps <quick_start_guide_service_apps>` to learn more.
+   - Its time to launch the app, connect your Android device to your PC and click **Run** on Android Studio toolbar.
+
+Step4: Navigating Through Sample App
+   Sample app comprises of 5 screens, described below:
+      
+   .. figure:: /_static/coco_sessions_flow.png
+      :align: center
+
+   - LoginActivity
```
Comment (manavkumarm) [inline source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/sample_app_walkthrough_android_media_client_apps.rst]: Provide a screen shot for each screen

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst (hunk 1 above)
Comment (elear-akshay) [reply source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst:5]: done.

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/sample_app_walkthrough_android_media_client_apps.rst (hunk 2 above)
Comment (elear-akshay) [reply source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/sample_app_walkthrough_android_media_client_apps.rst]: added.

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst [hunk 3]
```diff
@@ -5,10 +5,79 @@
...
+   - Its time to launch the app, connect your Android device to your PC and click **Run** on Android Studio toolbar.
+
+Step4: Navigating Through Sample App
+   Sample app comprises of 5 screens, described below:
+      
+   .. figure:: /_static/coco_sessions_flow.png
+      :align: center
+
+   - LoginActivity
+      This screen demonstrates the Third-party login solution for service apps.
```
Comment (manavkumarm) [inline source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst]: This section assumes that the user has already read the sample app walkthrough which comes after this section. Change the activity descriptions such that they are from the end user's point of view e.g. 

"This screen allows the user to login with his COCO credentials."

"This screen allows the user to view the Media Sessions that he is a part of."

...and so on

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst (hunk 3 above)
Comment (elear-akshay) [reply source/getting_started/zero_lag_media_edge/coco_for_media_applications/end_user_media_client_applications/android_end_user_media_client_applications/quick_start_guide_android_media_client_apps.rst]: improved as per suggestion.

**Review by manavkumarm — APPROVED** [review]

**Pull request merged**